lib/registration.pm

```diff
@@ -191,6 +193,42 @@ sub register_product {
     assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE');
 }
 
+=head2 register_product_proxyscc
+ 
+    register_product_proxyscc();
+
+Wrapper for SUSEConnect --url <scc_proxy_url> -r <regcode>.
+Requires SCC_URL and SCC_REGCODE variable.
+=cut
+sub register_product_proxyscc {
+    assert_script_run 'SUSEConnect --url ' . get_required_var('SCC_URL') . ' -r ' . get_required_var('SCC_REGCODE');
```

Comment:
Please try to use add_suseconnect_product to replace the line, if it works the function register_product_proxyscc can be removed.

Reply:
I checked code:
tests/console/suseconnect_scc.pm:    assert_script_run "SUSEConnect --url $scc_url -r $reg_code";
tests/console/zypper_info.pm:            assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE') . ' --url https://scc.suse.com';
Others use this command like up, I think it's better to create this function for common using. As it already has register_product API.

Reply:
Changed as other script:  assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE') . ' --url https://scc.suse.com';